cleanlab/segmentation/filter.py

```diff
@@ -0,0 +1,210 @@
+# Copyright (C) 2017-2023  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+
+"""
+Methods to find label issues in semantic segmentation datasets, where each pixel in a image receives its own class label.
+
+"""
+
+from cleanlab.experimental.label_issues_batched import find_label_issues_batched
+import numpy as np
+from typing import Tuple
+
+
+def find_label_issues(
+    labels: np.ndarray,
+    pred_probs: np.ndarray,
+    *,
+    downsample: int = 16,
+    batch_size: int = 10000,
+    verbose: bool = True,
+    **kwargs,
+) -> np.ndarray:
+    """
+    Returns a boolean mask for the entire dataset, per pixel where ``True`` represents
+    an example identified with a label issue and ``False`` represents an example of a pixel correctly labeled.
+
+    N - Number of images in the dataset
+    K - Number of classes in the dataset
+    H - Height of each image
+    W - Width of each image
+
+    Tip: if you encounter the error "pred_probs is not defined", try setting
+    ``n_jobs=1``.
+
+    Parameters
+    ----------
+    labels : np.ndarray
+      A discrete array of shape ``(N,H,W,)`` of noisy labels for a classification dataset, i.e. some labels may be erroneous.
+      *Format requirements*: for a dataset with K classes, each pixel must be integer in 0, 1, ..., K-1.
+
+    Tip: If your labels are one hot encoded you can `np.argmax(labels_one_hot,axis=1)` assuming that `labels_one_hot` is of dimension (N,K,H,W)
+    before entering in the function
```

Comment:
```suggestion
    Tip: If your labels are one hot encoded you can do: ``labels = np.argmax(labels_one_hot,axis=1)`` assuming that `labels_one_hot` is of dimension ``(N,K,H,W)``, in order to get properly formatted `labels`
```

Reply:
always use double back-ticks for code snippets in docstrings. single backtick is for referencing argument variables, see numpy documentation instructions